setup.py

```diff
@@ -1,48 +1,98 @@
 from __future__ import absolute_import, division, print_function
 
-from setuptools import setup, find_packages, Extension
+from distutils.command.sdist import sdist as _sdist
+import glob
 import os
 
-import numpy as np
-
-from Cython.Build import cythonize
+from setuptools import setup, find_packages, Extension
+from setuptools.command.build_ext import build_ext as _build_ext
 
+PACKAGE_NAME = 'stratify'
+PACKAGE_DIR = os.path.abspath(os.path.dirname(__file__))
+CYTHON_DIRECTIVES = {'linetrace': True, 'binding': True}
 
-NAME = 'stratify'
-DIR = os.path.abspath(os.path.dirname(__file__))
+try:
+    # Detect if Cython is available. Where it is, use it, otherwise fall back
+    # to C source included with source distribution.
+    import Cython
```

Comment:
This is intentional as a check for whether Cython is available or not.

Reply:
@tjtg We can circumvent the fussiness of `sticker` if you happy to adopt the following pattern:
```python
try:
    from Cython.Build import cythonize
except ImportError:
    cythonize = None
```
Then replace the use of `USE_CYTHON` with `cythonize` below as a logic switch, as either `cythonize` will refer to the actual Cython function on successful import (hence a truthy value) otherwise, we've set `cythonize` to a suitable falsy value.